ui/app/components/secret-engine/configuration-details.hbs

```diff
@@ -6,13 +6,14 @@
 {{#if @configModels.length}}
   {{#each @configModels as |configModel|}}
     {{#each configModel.attrs as |attr|}}
-      {{#if attr.options.sensitive}}
+      {{! public key while not sensitive when editing/creating, should be hidden by default on viewing }}
```

Comment:
Do we know for sure that it needs to be hidden on read? I remember we were masking PKI certificates for a while when that data isn't actually sensitive and doesn't need to be obfuscated.

Reply:
Great point. The current functionality is to hide. And I remember way back when we implemented the masked input features, we opted to mask all certificates—here is the [original PR](https://github.com/hashicorp/vault/pull/11284/files). Because I'm trying to keep behavior consistent, I will keep this masked because it's the current behavior, but I'll ask design as a follow up what the standard should be.
![image](https://github.com/user-attachments/assets/3e039269-bd92-43de-b180-f17af289787b)